lib/ansible/modules/net_tools/nsupdate.py

```diff
@@ -190,6 +190,11 @@ def __init__(self, module):
             if self.zone[-1] != '.':
                 self.zone += '.'
 
+        if module.params['record'][-1] != '.':
```

Comment:
seems unrelated to this PR

Reply:
How do you suggest I do the TTL lookup without having a fqdn to use for the lookup?

Reply:
i see